sql/core/src/test/scala/org/apache/spark/sql/InlineTableParsingImprovementsSuite.scala

```diff
@@ -226,4 +287,72 @@ class InlineTableParsingImprovementsSuite extends QueryTest with SharedSparkSess
       }
     }
   }
+
+  test("Value list in subquery") {
```

Comment:
Let's add a JIRA prefix in the test title

```suggestion
  test("SPARK-49269: Value list in subquery") {
```